src/diagnostics/defaults/moisture_model.jl

```diff
@@ -0,0 +1,19 @@
+# FIXME: Gabriele added this as an example. Put something meaningful here!
+function get_default_diagnostics(::T) where {T <: DryModel}
```

Comment:
```suggestion
function get_default_diagnostics(::DryModel)
```
We don't need to force specialization on this

Reply:
Right